Project1_2015.py

```diff
@@ -1,15 +1,15 @@
 with open('Project1_2015', 'r') as file:
     current_file = file.read()
 floor_counter = 0
-basement_position = 0
+basement_position = None
 
 for index, char in enumerate(current_file):
     if char == '(':
         floor_counter += 1
     else:
         floor_counter -= 1
 
-    if floor_counter == -1 and basement_position == 0:
+    if basement_position is None and floor_counter == -1:
```

Comment:
checking `is None` first avoids `floor_counter ==  -1` check if `basement_position` is already not `None`

Reply:
why is one check better than the other? I thought checking 1 variable is equal to checking another.

Reply:
http://jaredgrubb.blogspot.com/2009/04/python-is-none-vs-none.html

Reply:
never mind, I get it